test/quantization/test_quantize_script.py

```diff
@@ -2559,13 +2559,51 @@ def forward(self, x):
         qconfig_dict = {'': default_dynamic_qconfig}
         m1 = quantize_dynamic_script(model, qconfig_dict)
         out_graph = m1(data)
-
-        m2 = prepare_dynamic_script(model, qconfig_dict)
-        m2(data)
-        m2 = convert_dynamic_script(m2)
-        out_ref = m2(data)
-        self.assertEqual(out_graph, out_ref)
-
         FileCheck().check_count("quantized::linear_dynamic(", 2, exactly=True) \
                    .check_not("aten::_choose_qparams_per_tensor") \
                    .run(m1.graph)
+
+        # Check to make sure weight observers run correctly
+        ref_qparams = []
+        qconfig = script_qconfig(default_dynamic_qconfig)
+        for wt in [model.res1.weight, model.res2.weight]:
+            get_forward(qconfig.weight)(wt)
+            qparams = qconfig.weight._get_method('calculate_qparams')()
```

Comment:
I think you can call them directly, e.g. `qconfig.weight(wt)` and `qconfig.weight.calculate_qparams`

Reply:
It doesn't seem to be directly callable `TypeError: 'torch._C.ScriptModule' object is not callable`

Reply:
I see, maybe you can wrap it into a RecursiveScriptModule with `wrap_cpp_module` first